my-app/src/components/Posts.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import Post from "./Post.js";
+import "../App.css";
+
+class Posts extends React.Component {
+  constructor() {
+    super();
+  }
+  render() {
+    let currentPage = this.props.currentPage;
+    return (
+      <ol className="posts">
```

Comment:
Looks like invalid HTML because ordered and unordered lists need to contain `<li/>` but `<Post/>` is a div